src/transition.js

```diff
@@ -61,10 +61,11 @@ export default class Transition {
 
     if (currentTweetIndex === this.numOfActualTweets) {
       this._finishedTransition();
```

Comment:
I think we should fade out the last tweet with `$(previousTweet).fadeOut(this.fadeTime);`. It's very abrupt right now. 

Also maybe set `this._setCurrentTweetIndex `to 0 here. Noticed you removed it.

Reply:
@jungeunyoon I think it will fade out the last tweet because this.numOfActualTweets is always greater than currentTweetIndex by one. I tested and that is how it is working. Do you see something different? cheers

Reply:
@jungeunyoon regarding the _setCurrentTweetIndex to 0 it will be done by the finishTransition. cheers